specification/azuredependencymap/resource-manager/Microsoft.DependencyMap/preview/2024-06-01-preview/dependencyMap.json

```diff
@@ -0,0 +1,1444 @@
+{
```

Comment:
Why the new swagger spec?

Reply:
yeah my bad just fixed it, copied the change I made from RPSaaSMaster there and it had this extra version originally